gio/src/list_store.rs

```diff
@@ -52,6 +64,28 @@ impl<O: IsA<ListStore>> ListStoreExtManual for O {
             )
         }
     }
+
+    fn find_with_equal_func<P: IsA<glib::Object>, F: Fn(&Object, &Object) -> glib::ffi::gboolean>(
+        &self,
+        item: &P,
+        equal_func: F,
+    ) -> Option<u32> {
+        unsafe {
+            let mut position = mem::MaybeUninit::uninit();
```

Comment:
```suggestion
            let mut position = std::mem::MaybeUninit::uninit();
```
and drop the mem import you have at the top of the file